psi4/driver/procrouting/proc.py

```diff
@@ -786,6 +786,30 @@ def select_ccsd_gradient(name, **kwargs):
     else:
         return func(name, **kwargs)
 
+def select_cc2_gradient(name, **kwargs):
```

Comment:
I'm a little confused over why do a select fn for cc2 gradient when they can only be generated by one module (cc*) and there isn't one for cc2 energy. "select" fns are generally for the purpose of routing calculations through the code that should take different paths depending on reference, df-ness, etc. I'm perfectly fine with having a select fn for cc2, especially if there'll be further variations on it, and I can fill them in myself (needs a block [here](https://github.com/psi4/psi4/blob/master/doc/sphinxman/source/notes_c.rst), too), I just want to make sure it's deliberate and consistent.